GOVERNANCE.md

```diff
@@ -48,12 +48,9 @@ The **Project Governance Board** (**PGB**)'s responsibility is to ensure that ev
 The **PGB** also follows and is responsible for upholding the [OASIS Open Projects Rules](https://github.com/oasis-open-projects/documentation/blob/master/board-docs/open-projects-rules.md).
 
 
-### List of PGB Members
+### PGB Membership
 
-* Daniel Burnett (@burnburn) - ConsenSys
-* Chaals Nevile (@chaals) - EEA
-* Tas Dienes (@tasdienes) - EF
-* Jory Burson (@jorydotcom / @OASIS-OP-Admin) - OASIS Staff
```

Comment:
Can we put this in the README.md, rather than the governance doc?

(I don't mind either way, but the README solution feels a bit cleaner to me, and better meeting the requirement. Although your mileage may vary)

Reply:
Agreed.  It doesn't belong in the general governance rules doc.  The "General Information web page" would, in fact, be the README.md.

Reply:
The names are in the README.md.  Marking as resolved.